IdokladSdk.UnitTests/Tests/Authentication/Extensions/RestClientExtensionsTests.cs

```diff
@@ -30,6 +33,16 @@ public void RequestAuthorizationCodeToken_ServiceUnavailable_Throws()
             Assert.Throws<IdokladUnavailableException>(() => _serviceUnavailableClient.RequestAuthorizationCodeToken(request));
         }
 
+        [Test]
```

Comment:
Asi nebudu ten PR blokovat, ale váhal jsem, jestli ty testy async metod nedát do samostatného zdrojáku RestClientExtensionsTests.Async.cs. Byly by ve stejné třídě (partial), jen ve dvou zdrojácích. Máme tak dělané i testy na jednotlivé klienty, vlastně i ten RestClientExtensions je partial třída rozdělená do dvou zdrojáků. Tak kdyby se Ti chtělo to ještě opravit, tak na to zadej další PR, tento schválím.

Reply:
rozdelené